modelator/pytest/decorators.py

```diff
@@ -50,7 +72,7 @@ def itf(filepath: str, keypath=None):
 
     def decorator(func: Callable) -> Callable:
         with open(filepath, encoding="utf-8") as f:
-            trace = json.load(f)["states"]
+            trace = munchify(sanitize_itf(json.load(f)["states"]))
```

Comment:
I added `munchify` for convenience at `@step` decorated methods. Now we can write `param.key` instead of `param["key"]`.